src/parser/mod.rs

```diff
@@ -7979,6 +7979,27 @@ impl<'a> Parser<'a> {
         }
     }
 
+    pub fn parse_enum_values(&mut self) -> Result<Vec<EnumValue>, ParserError> {
+        self.expect_token(&Token::LParen)?;
+        let values = self.parse_comma_separated(Parser::parse_enum_value)?;
+        self.expect_token(&Token::RParen)?;
+        Ok(values)
+    }
+
+    pub fn parse_enum_value(&mut self) -> Result<EnumValue, ParserError> {
+        let str = self.parse_literal_string()?;
+        let value = match self.peek_token().token {
```

Comment:
This can be simplified as
```rust
if self.consume_token(&Token::Eq) {
    Enum::Pair()
} else {
    Enum::String()
}
```
one other more of a nit, since the method is small enough and if its only used once, it could probably be inlined within the `parse_enum_values()`

Reply:
Good point.